titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/V3GrpcModelConverters.java

```diff
@@ -555,10 +591,37 @@ public static TaskState toCoreTaskState(com.netflix.titus.grpc.protogen.TaskStat
         return builder.build();
     }
 
+    public static AddressLocation toGrpcAddressLocation(IpAddressLocation coreIpAddressLocation) {
+        return AddressLocation.newBuilder()
+                .setAvailabilityZone(coreIpAddressLocation.getAvailabilityZone())
+                .setRegion(coreIpAddressLocation.getRegion())
+                .setSubnetId(coreIpAddressLocation.getSubnetId())
+                .build();
+    }
+
+    public static AddressAllocation toGrpcAddressAllocation(IpAddressAllocation coreIpAddressAllocation) {
+        return AddressAllocation.newBuilder()
+                .setUuid(coreIpAddressAllocation.getAllocationId())
+                .setAddress(coreIpAddressAllocation.getIpAddress())
+                .setAddressLocation(toGrpcAddressLocation(coreIpAddressAllocation.getIpAddressLocation()))
+                .build();
+    }
+
+    public static SignedAddressAllocation toGrpcSignedAddressAllocation(SignedIpAddressAllocation coreSignedIpAddressAllocation) {
+        return SignedAddressAllocation.newBuilder()
+                .setAddressAllocation(toGrpcAddressAllocation(coreSignedIpAddressAllocation.getIpAddressAllocation()))
+                .setSignedAddressAllocation(ByteString.copyFrom(coreSignedIpAddressAllocation.getIpAddressAllocationSignature()))
+                .build();
+    }
+
     public static com.netflix.titus.grpc.protogen.ContainerResources toGrpcResources(ContainerResources containerResources) {
         List<com.netflix.titus.grpc.protogen.ContainerResources.EfsMount> grpcEfsMounts = containerResources.getEfsMounts().isEmpty()
                 ? Collections.emptyList()
                 : containerResources.getEfsMounts().stream().map(V3GrpcModelConverters::toGrpcEfsMount).collect(Collectors.toList());
+        List<SignedAddressAllocation> grpcSignedAddressAllocation = containerResources.getSignedIpAddressAllocations().isEmpty()
+                ? Collections.EMPTY_LIST
```

Comment:
`Collections.EMPTY_LIST` -> `Collections.emptyList()` for consistency

Reply:
Fixed.